core/resharding.go

```diff
@@ -24,7 +24,7 @@ const (
 	// GenesisShardNum is the number of shard at genesis
 	GenesisShardNum = 4
 	// GenesisShardSize is the size of each shard at genesis
-	GenesisShardSize = 50
+	GenesisShardSize = 5
```

Comment:
don't change the default value.

Reply:
This is probably a leak from testing (I have the same change in my local).  Below please find/accept the revert.

```suggestion
	GenesisShardSize = 50
```

Reply:
Test code reverted.